src/components/dashboard/Dashboard.js

```diff
@@ -393,26 +393,15 @@ const Dashboard = props => {
     // Android never gets values from measure causing animation to crash because of NaN
     const width = measurements.width || 0
 
-    const balanceCenteredPosition = headerContentWidth / 2 - width / 2
-
     setBalanceBlockWidth(width)
-
-    Animated.timing(headerBalanceRightAnimValue, {
-      toValue: balanceCenteredPosition,
-      duration: 50,
-    }).start()
-
-    if (!showBalance) {
-      setShowBalance(true)
-    }
-  }, [setBalanceBlockWidth, showBalance, setShowBalance, headerContentWidth, headerBalanceRightAnimValue])
+  }, [setBalanceBlockWidth])
 
   useEffect(() => {
     const timing = 250
     const fullNameOpacityTiming = 150
     const easingIn = Easing.in(Easing.quad)
     const easingOut = Easing.out(Easing.quad)
-    const balanceCenteredPosition = headerContentWidth / 2 - balanceBlockWidth / 2
+    const balanceCalculatedLeftMargin = headerContentWidth - balanceBlockWidth - 20
```

Comment:
add comment why -20

Reply:
Added comment explaining that and also updated the comment on `saveBalanceBlockWidth` (Line 380)